js/ccex.js

```diff
@@ -131,6 +133,48 @@ module.exports = class ccex extends Exchange {
         return this.parseOrderBook (orderbook, undefined, 'buy', 'sell', 'Rate', 'Quantity');
     }
 
+    async fetchOrderBooks(symbols = undefined, params = {}){
+        await this.loadMarkets ();
+        let orderBooks = [];
+        let orderBooksResult = [];
+
+        try {
+            let response = await this.publicGetFullorderbook();
+            if(response && response.success && response.result) {
+                for (let type of Object.keys(response.result)) {
+                    for (let order of response.result[type]) {
+                        let index = orderBooks.findIndex(function (f){
+                            return f.symbol === order.Market.replace("-", "/").toUpperCase()
```

Comment:
Assoc keys should be single quoted `f['symbol']` = good, `f.symbol` = bad (won't transpile).